apps/storybook/src/Tabs.stories.tsx

```diff
@@ -32,210 +25,285 @@ export default {
-        <Tab key={3} label='Item3' />,
-      ]}
-      {...args}
-      onTabSelected={setIndex}
-    >
-      {getContent()}
+    <Tabs {...args}>
+      <Tabs.TabList>
+        <Tabs.Tab>
+          <Tabs.TabInfo>
+            <Tabs.TabLabel>Item1</Tabs.TabLabel>
+          </Tabs.TabInfo>
+        </Tabs.Tab>
+        <Tabs.Tab>
+          <Tabs.TabInfo>
+            <Tabs.TabLabel>Item2</Tabs.TabLabel>
+          </Tabs.TabInfo>
+        </Tabs.Tab>
+        <Tabs.Tab>
+          <Tabs.TabInfo>
+            <Tabs.TabLabel>Item3</Tabs.TabLabel>
+          </Tabs.TabInfo>
+        </Tabs.Tab>
+      </Tabs.TabList>
+
+      <Tabs.Actions>
+        <Tabs.Action>
+          <Button key={'Small'} size={'small'}>
+            Small size button
+          </Button>
+          <Button key={'Normal'}>Normal size button</Button>
+        </Tabs.Action>
+      </Tabs.Actions>
+
+      <Tabs.Panels>
+        <Tabs.Panel>
+          Bentley Systems, Incorporated, is an American-based software
+          development company that develops, manufactures, licenses, sells and
+          supports computer software and services for the design, construction,
+          and operation of infrastructure. The company&apos;s software serves
+          the building, plant, civil, and geospatial markets in the areas of
+          architecture, engineering, construction (AEC) and operations. Their
+          software products are used to design, engineer, build, and operate
+          large constructed assets such as roadways, railways, bridges,
+          buildings, industrial plants, power plants, and utility networks. The
+          company re-invests 20% of their revenues in research and development.
+        </Tabs.Panel>
+        <Tabs.Panel>
+          Bentley Systems is headquartered in Exton, Pennsylvania, United
+          States, but has development, sales and other departments in over 50
+          countries. The company had revenues of $700 million in 2018.
+        </Tabs.Panel>
+        <Tabs.Panel>
+          Keith A. Bentley and Barry J. Bentley founded Bentley Systems in 1984.
+          They introduced the commercial version of PseudoStation in 1985, which
+          allowed users of Intergraphs VAX systems to use low-cost graphics
+          terminals to view and modify the designs on their Intergraph IGDS
+          (Interactive Graphics Design System) installations. Their first
+          product was shown to potential users who were polled as to what they
+          would be willing to pay for it. They averaged the answers, arriving at
+          a price of $7,943. A DOS-based version of MicroStation was introduced
+          in 1986.
+        </Tabs.Panel>
+      </Tabs.Panels>
     </Tabs>
   );
 };
 DefaultTabs.args = {
   type: 'default',
-  labels: [
-    <Tab key={1} label='Item1' />,
-    <Tab key={2} label='Item2' />,
-    <Tab key={3} label='Item3' />,
-  ],
 };
 
 export const BorderlessTabs: Story<Partial<TabsProps>> = (args) => {
```

Comment:
Some of the visual tests aren't passing due to the stripe not extending all the way to the edge of the tab on the initial render. Seems to happen with all borderless examples, but most noticeable in these two stories. This needs to be fixed. 

![image](https://github.com/iTwin/iTwinUI/assets/78564221/30f94cc8-9277-462e-b6f7-13728701ff29)

![image](https://github.com/iTwin/iTwinUI/assets/78564221/5037e306-3022-4bb0-bfae-e64b5a84f0f6)

Reply:
Agree, this needs to be fixed.

Reply:
This has been fixed by adding a timeout of 50ms so that the animation can finish before the stripe is applied. Let me know if there's a better solution for this

Reply:
As @mayank99 mentioned in [this comment](https://github.com/iTwin/iTwinUI/pull/1401#issuecomment-1640609214), setting a timeout is not the proper solution here.

Reply:
This issue has now been fixed by setting the stripe properties inside of `Tabs.TabList` and using `setStripeProperties` to set the properties in `Tabs`